react-state/my-app/src/ImageCaption.tsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+
+type Props = {
+  imageCap: string[];
+};
+
+export function ImageCaption({ imageCap }: Props) {
+  const [Index, setCurrentIndex] = useState(0);
```

Comment:
State variables are still variables, so they should use variable naming conventions -- start with lowercase.